### ISSUE TITLE
dropped frames when playing 60FPS videos

### ISSUE THREAD

**Issue by YousefSaber** [mpv-player/mpv#9420]
my GPU is Intel HD Graphics 620 

mpv version info 
```
(base) [josephsaber@fedora ~]$ mpv --version
mpv 0.33.1 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
FFmpeg library versions:
   libavutil       56.70.100
   libavcodec      58.134.100
   libavformat     58.76.100
   libswscale      5.9.100
   libavfilter     7.110.100
   libswresample   3.9.100
FFmpeg version: 4.4.1
```
Fedora 35 workstation 

when playing 60FPS videos the player shows me a message when checking its status using `i` that there are dropped frames 
[problem.txt](https://github.com/mpv-player/mpv/files/7493164/problem.txt)

here's a log file

**Comment by ValZapod** [comment]
Your display is 60.011000 fps while the video is 60/1.001 fps. No surprises here. Try this, it is 60.000 fps: https://www.youtube.com/watch?v=F845MOydfec

Guys, is there any command that will show decodinb drops, not presentation drops?

**Comment by YousefSaber** [comment]
@ValZapod MPV still drops frames despite playing the video, to be honest with you I can't notice any noticeable lag with my eye when watching the video, as you say this may be normal behavior and H/W acceleration just works fine you may want to close the issue if you want.
[60fpsproblem.txt](https://github.com/mpv-player/mpv/files/7512837/60fpsproblem.txt)

**Comment by Dudemanguy** [comment]
Is this still a problem?

**Comment by cold-nagai** [comment]
> Is this still a problem?

Yes. I'm using mpv on Wayland, with vaapi hwdec via libva-intel-driver and this problem still occurs.

**Comment by simontretter** [comment]
Same issue here on Intel(R) Xe Graphics (TGL GT2)

`mpv 0.36.0 Copyright © 2000-2023 mpv/MPlayer/mplayer2 projects
 built on Sun Jul 23 05:49:29 2023
libplacebo version: v5.264.1
FFmpeg version: n6.0
FFmpeg library versions:
   libavutil       58.2.100
   libavcodec      60.3.100
   libavformat     60.3.100
   libswscale      7.1.100
   libavfilter     9.3.100
   libswresample   4.10.100`
